src/storage.rs

```diff
@@ -427,18 +427,72 @@ impl Store {
     }
 
     /// Fetch foreign audits, only call this is we're not --locked
-    pub async fn fetch_foreign_audits(&mut self, network: &Network) -> Result<(), FetchAuditError> {
-        let new_imports = ImportsFile {
-            audits: try_join_all(self.config.imports.iter().map(|(name, import)| async {
+    pub async fn fetch_foreign_audits(
+        &mut self,
+        network: &Network,
+        accept_changes: bool,
+    ) -> Result<(), FetchAuditError> {
+        let raw_new_imports =
+            try_join_all(self.config.imports.iter().map(|(name, import)| async {
                 let audit_file = fetch_foreign_audit(network, name, &import.url).await?;
-                Ok::<_, FetchAuditError>((name.clone(), audit_file))
+                // Fetch the descriptions to cache them and check that they haven't changed
+                // FIXME: this should probably treat failing to fetch as an error but eula_for_criteria
+                // hides errors... should we have two versions? Or make it the caller's problem?
```

Comment:
Probably should make it the caller's problem, and then perhaps add a special infallible version?

Reply:
Gonna leave this to a followup